src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -491,8 +491,9 @@ internal string GetIntputOutputTrace(string rawInputOutputData)
             }
         }
 
-        // Get a response that will point to our webhook handler.
-        internal HttpResponseMessage CreateCheckStatusResponse(
+
+        // Get a response that will point to our webhook handler. 
+        protected internal virtual HttpResponseMessage CreateCheckStatusResponse(

```

Comment:
Why was it necessary to make this `protected` and `virtual`? I can't think of a good reason to expose anything in  `DurableTaskExtension`.

Reply:
This was done in order to override the behavior in the mock class. I will revert these changes based on your feedback. Thank you!